src/RequestHandle.cpp

```diff
@@ -389,6 +424,16 @@ auto RequestHandle::setCompletionStatus(
         case CURLcode::CURLE_SSL_CONNECT_ERROR:
             m_status_code = RequestStatus::CONNECT_SSL_ERROR;
             break;
+        case CURLcode::CURLE_WRITE_ERROR:
+            /**
+             * If there is a cURL write error, but the maximum number of bytes has been written,
+             * then we intentionally aborted, so let's set this to success.
+             * Otherwise, we have a bug and downloaded too many bytes.
+             */
+            m_status_code = (getRemainingDownloadBytesOptional())
+                            ? RequestStatus::SUCCESS
```

Comment:
whitespace

Reply:
I wasn't sure if this is what you had in mind -- I couldn't find another example in the code.